recipe/meta.yaml

```diff
@@ -19,10 +19,12 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
```

Comment:
Does this also need to be in host?

Reply:
Maybe, I copied this from the qt package (since I thought they would know of all people) here: https://github.com/conda-forge/qt-feedstock/blob/master/recipe/meta.yaml#L35